core/iwasm/libraries/thread-mgr/thread_manager.h

```diff
@@ -51,6 +51,13 @@ struct WASMCluster {
 #if WASM_ENABLE_DEBUG_INTERP != 0
     WASMDebugInstance *debug_inst;
 #endif
+
+#if WASM_ENABLE_DUMP_CALL_STACK != 0
+    /* When an exception occurs in a thread, the stack frames of that thread are
+     * saved into the claster
+     */
+    Vector *exception_frames;
```

Comment:
Does it have to be a pointer or can it be a value? We could reduce a number of heap allocations (not much the actual size itself though) and simplify the initialization code.